packages/meteor-wallet/src/lib/meteor-wallet.ts

```diff
@@ -37,6 +37,7 @@ const setupWalletState = async (
     headers: {},
   });
 
+  // @ts-ignore
   const wallet = new MeteorWalletSdk({ near, appKeyPrefix: "near_app" });
```

Comment:
What's the reason for `@ts-ignore` here?

Reply:
> What's the reason for `@ts-ignore` here?

![image](https://user-images.githubusercontent.com/95852708/179939295-c18d8e63-180f-4e94-a29b-90f86c133dcf.png)